src/Core/src/Platform/Tizen/ImageSourcePartExtensions.cs

```diff
@@ -1,30 +1,41 @@
 ﻿using System;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
 using NView = Tizen.NUI.BaseComponents.View;
 using TImage = Tizen.NUI.BaseComponents.ImageView;
 
 namespace Microsoft.Maui.Platform
 {
 	public static class ImageSourcePartExtensions
 	{
-		public static async Task<IImageSourceServiceResult<MauiImageSource>?> UpdateSourceAsync(this IImageSourcePart image, NView destinationContext, IImageSourceServiceProvider services, Action<MauiImageSource?> setImage, CancellationToken cancellationToken = default)
```

Comment:
@PureWeen Was it necessary to match the API here? Tizen doesn't looks like it uses a scale. I jsut wanted to be consistent.